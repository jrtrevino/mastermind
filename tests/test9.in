f
231
zz
A
...
2
.
0
1
AA
