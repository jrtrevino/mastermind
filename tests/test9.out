Enter the seed for random number generation: Seed must be an integer value, please try again
Enter the seed for random number generation: Enter the maximum letter for the game (A-Z): The letter must be an uppercase A-Z, please try again
Enter the maximum letter for the game (A-Z): Enter the number of positions for the game (1-8): The number of positions must be 1-8, please try again
Enter the number of positions for the game (1-8): Enter the number of guesses allowed for the game: The number of guesses must be a positive integer, please try again
Enter the number of guesses allowed for the game: The number of guesses must be a positive integer, please try again
Enter the number of guesses allowed for the game: 
Enter guess 1: 
Wow, you won in 1 guesses - well done!
